src/pyodide.js

```diff
@@ -1,37 +1,136 @@
+// Regexps for validating package name and URI
+const NAME_REGEXP = '[a-z0-9_][a-z0-9_\-]*'
```

Comment:
We shouldn't add more than we have to to the global namespace.  These should remain as internal variables, unless we really need to add them to the public API (I don't think we do).

Reply:
I agree, I'm thinking in terms of module-based JavaScript, where this would by module local. I'm hoping you'll take a look at this repo (https://github.com/darthtrevino/pyodide-loader) and I can transfer it to you guys. I pushed it to npm this morning to reserve the name, and I'd be glad to give you the keys to that as well

Reply:
I just added a rollup step to the pyodide-loader repo, it will install the languagePluginLoader as a function on the window, which should make it map closer to the original ergonomics of it.